helm/prometheus-meta-operator/templates/alertmanager/alertmanager-ingress.yaml

```diff
@@ -2,7 +2,12 @@ apiVersion: networking.k8s.io/v1
 kind: Ingress
 metadata:
   annotations:
-    kubernetes.io/tls-acme: "{{ .Values.prometheus.letsencrypt }}"
+    {{- if .Values.prometheus.letsencrypt }}
```

Comment:
We have a few ingresses that are generated in PMO code. Could you also add it to them?

Reply:
Oh right because they rely on the main ones as they are subpaths